src/test/scala/org/constellation/util/MajorityStateChooserTest.scala

```diff
@@ -115,6 +115,25 @@ class MajorityStateChooserFreeTest extends FreeSpec with Matchers {
         result._1 shouldBe List(6, 4, 2, 0).map(toRecentSnapshot)
         result._2 shouldBe Set(Id("node1"), Id("node2"))
       }
+
+      "chooses correct majority when encountering a 1/3 split" in {
```

Comment:
it would be great to describe the logic a little bit here, like:
```
"when encountering a 1/3 split" - {
 "should choose majority snapshot based on ..." in {
  ///
 }
}
```

That way the unit tests are becoming a live specification